packages/cloud-cognitive/src/components/EditTearsheet/index.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright IBM Corp. 2021, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export { EditTearsheet } from './EditTearsheet';
+export { EditTearsheetStep } from './EditTearsheetStep';
+export { EditTearsheetDivider } from './EditTearsheetDivider';
```

Comment:
i think this is where the build error is coming from. i think you may have forgotten to add this file to the git push.